client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -29,50 +23,34 @@ class MediaModalSecondaryActions extends Component {
 		user: PropTypes.object,
 		site: PropTypes.object,
 		selectedItems: PropTypes.array,
-		view: PropTypes.oneOf( values( ModalViews ) ),
-		disabled: PropTypes.bool,
 		onDelete: PropTypes.func,
 		onViewDetails: PropTypes.func,
 	};
 
 	static defaultProps = {
-		disabled: false,
 		onDelete: noop,
 	};
 
 	getButtons() {
-		const {
-			disabled,
-			selectedItems,
-			site,
-			translate,
-			user,
-			view,
-
-			onDelete,
-			onViewDetails,
-		} = this.props;
+		const { selectedItems, site, translate, user, onDelete, onViewDetails } = this.props;
 
 		const buttons = [];
 
-		if ( ModalViews.LIST === view && selectedItems.length ) {
+		if ( selectedItems.length ) {
 			buttons.push( {
 				key: 'edit',
 				text: translate( 'Edit' ),
-				disabled: disabled,
 				primary: true,
 				onClick: onViewDetails,
 			} );
 		}
 
 		const canDeleteItems =
 			selectedItems.length &&
-			every( selectedItems, ( item ) => {
-				return canUserDeleteItem( item, user, site );
-			} );
+			selectedItems.every( ( item ) => canUserDeleteItem( item, user, site ) );
 
-		if ( ModalViews.GALLERY !== view && canDeleteItems ) {
-			const isButtonDisabled = disabled || some( selectedItems, 'transient' );
+		if ( canDeleteItems ) {
+			const isButtonDisabled = selectedItems.some( ( item ) => item.transient );
```

Comment:
We could simplify here by destructuring:

```suggestion
			const isButtonDisabled = selectedItems.some( ( { transient } ) => transient );
```

Reply:
I wouldn't say this is a simplification -- the destructured version is four characters longer 🙂

Reply:
Simplification !== Shortification 😉